### PR TITLE
document pagination for varsets

### PR DESCRIPTION
**Pull request by SwiftEngineer**
### What
Add information on how to paginate varsets.

### Why
It's unclear whether or not the documentation was created before/after the pagination was added, but we do know this functionality has actually been in TFC for quite some time, but for some reason was never actually documented.

In any case, this PR should fill the gap!

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Review by mpminardi — APPROVED** [review]
Looks good to me!

**Review by mgarrell777 — APPROVED** [review]
Hi @SwiftEngineer I'm Matthew, new technical writer for Terraform. I approve with a couple rewording suggestions. Thanks very much.

File: website/docs/cloud-docs/api-docs/variable-sets.mdx [hunk 1]
```diff
@@ -309,6 +309,15 @@ List all variable sets for a workspace.
 | --------------- | ---------------- |
 | `:workspace_id` | The workspace ID |
 
+### Query Parameters
+
+Both of the endpoints above support pagination [with standard URL query parameters](/cloud-docs/api-docs/#query-parameters). Remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/api-docs/variable-sets.mdx:314]: Suggest rewording to "Both of the endpoints support pagination [with standard URL query parameters](/cloud-docs/api-docs/#query-parameters). Remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling does not automatically encode URLs."
